dump-hr/internship-app
# RDY: Admin login

This PR closes #5

To test login functionality follow next steps:

1. Navigate to `/login` page
2. Input `admin@dump.hr`: `dump.1950` creds
3. Click on login button

To check that login is valid, you should be toasted with valid login message as well as redirected to home page with history being restacked.

To check unhappy case of login flow, try with some other creds pair and you should be toasted with invalid message and anchored to login page so you can repeat login flow.

Invalid mail format should be immediately warned as wrong.
